src/AspNetCoreModuleV2/CommonLib/aspnetcoreconfig.cxx

```diff
@@ -322,29 +320,18 @@ ASPNETCORE_CONFIG::Populate(
         }
     }
 
-    bstrWebsocketSection = SysAllocString(CS_WEBSOCKET_SECTION);
-    if (bstrWebsocketSection == NULL)
-    {
-        hr = E_OUTOFMEMORY;
-        goto Finished;
-    }
-
-    hr = pAdminManager->GetAdminSection(bstrWebsocketSection,
-        m_struConfigPath.QueryStr(),
-        &pWebSocketElement);
+    // Even though the applicationhost.config file contains the websocket element,
+    // the websocket module may still not be enabled. Check if the
+    PCWSTR pszVariableValue;
+    DWORD cbLength;
+    hr = pHttpContext->GetServerVariable("WEBSOCKET_VERSION", &pszVariableValue, &cbLength);
     if (FAILED(hr))
```

Comment:
`m_fWebSocketEnabled = !FAILED(hr)`

Reply:
oops I meant to fix that. I'll merge that to dev.